### PR TITLE
data:atanda0x

### PR DESCRIPTION
**Pull request by atanda0x**

### CODE REVIEW

**Review by Njong392 — APPROVED** [review]
Looks great! :100: Thank you for contributing!

**Review by EmmaDawsonDev — COMMENTED** [review]
Hi @atanda0x this PR is almost ready to merge. Let us know if you need any help making the requested changes.

File: data/atanda0x.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/atanda0x",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/nafiu-atanda-7685681a7/",
+      "icon": "FaLinkedIn"
```
Comment (priyankarpal) [inline data/atanda0x.json:16]: change the icon to `FaLinkedin`

File: data/atanda0x.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/atanda0x",
+      "icon": "FaGithub"
+    },
```
Comment (priyankarpal) [inline data/atanda0x.json:22]: Delete the ","

**Review by priyankarpal — APPROVED** [review]

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Change these

**Comment by SaraJaoude** [comment]
> Change these

@priyankarpal please try to be more friendly with your comments, it will scare people away from open source and our project

**Comment by SaraJaoude** [comment]
I will fix via a temporary branch and then merge into `main`

**Pull request merged**